starboard/android/apk/app/src/main/java/dev/cobalt/media/MediaCodecBridge.java

```diff
@@ -538,9 +538,8 @@ public void onOutputFormatChanged(MediaCodec codec, MediaFormat format) {
         };
     mMediaCodec.setCallback(mCallback, mHandler);
 
-    // TODO: support OnFrameRenderedListener for non tunnel mode
-    if (tunnelModeAudioSessionId != -1) {
-      mTunnelModeFrameRendererListener =
+    if (isFrameRenderedCallbackEnabled() || tunnelModeAudioSessionId != -1) {
```

Comment:
Consider moving the check `tunnelModeAudioSessionId != -1` to `isFrameRenderedCallbackEnabled()`.

Reply:
isFrameRenderedCallbackEnabled() is a static function, but tunnelModeAudioSessionId is not, and it's used before MediaCodecBridge is created.